src/converters/audio_converter.ml

```diff
@@ -99,3 +99,69 @@ module Samplerate = struct
            in
            f converters_conf#get))
 end
+
+module Channel_layout = struct
+  exception Unsupported
+  exception Invalid_data
+
+  type layout = [ `Mono | `Stereo | `Five_point_one ]
```

Comment:
It would be nice to add `` `Empty `` so that we can generate blank, which is a very very useful case (for video). This would however passing the length of data we want to generate around.

Reply:
Just added support for it! It'll be done with the decoders, not as channel layout converter. Thanks for pointing it out..